dcos/mesos.py

```diff
@@ -51,8 +51,11 @@ def get_dcos_url(self, path):
         """
         if self._dcos_url:
             return urllib.parse.urljoin(self._dcos_url, path)
+        elif self._mesos_master_url:
+            mesos_master_url = ':'.join(self._mesos_master_url.split(':')[:2])
+            return urllib.parse.urljoin(mesos_master_url + '/', path)
```

Comment:
Instead of this `elif` we should just remove [this if](https://github.com/issenec/dcos-cli/blob/25bdc9561b9850824c291ec6a96468ab837d626a/dcos/mesos.py#L37) and always set `self._dcos_url`.